griptape/tools/computer/tool.py

```diff
@@ -97,6 +94,8 @@ def execute_command(self, params: dict) -> BaseArtifact:
         return self.execute_command_in_container(command)
 
     def execute_command_in_container(self, command: str) -> BaseArtifact:
+        from docker.models.containers import Container
+
```

Comment:
should these be wrapped with `import_optional_dependency`?

Reply:
We don't use `import_optional_dependency` for Tools since it's expected that the Tool will install them.

Reply:
Not saying this is ideal, but it's consistent with how other Tools work